setup.py

```diff
@@ -178,7 +178,7 @@ def get_git_sha() -> str:
         "trino": ["trino>=0.324.0"],
         "prophet": ["prophet>=1.0.1, <1.1", "pystan<3.0"],
         "redshift": ["sqlalchemy-redshift>=0.8.1, < 0.9"],
-        "rockset": ["rockset>=0.8.10, <0.9"],
+        "rockset": ["rockset-sqlalchemy>=0.0.1"],
```

Comment:
@gadhagod would you mind including an upper bound (exclusive of) the next major version?

Reply:
Thanks for the review @john-bodley. I have included an upper bound of the next major version here: https://github.com/apache/superset/pull/24649/commits/f9fd42c576509a41471d7b7464d8839f25d157ec.